src/metatensor/models/cli/export.py

```diff
@@ -60,7 +64,10 @@ def export_model(model: Any, output: Union[Path, str] = "exported-model.pt") ->
     path = str(check_suffix(filename=output, suffix=".pt"))
 
     if is_exported(model):
+        logger.info(f"The model is already exported. Saving it to `{path}`.")
```

Comment:
Cool. I am always struggeling if we should also add a test if these infos are displayed... But I think it is fine for now.